bark/models/tests/behavior_idm_classic_test.cc

```diff
@@ -413,10 +400,7 @@ TEST(coolness_factor_lower_eq_case_vel_diff_pos, behavior_idm_classic) {
 
   // Create an observed world with specific goal definition and the
   // corresponding mcts state
-  Polygon polygon(
-      Pose(1, 1, 0),
-      std::vector<Point2d>{Point2d(0, 0), Point2d(0, 2), Point2d(2, 2),
-                           Point2d(2, 0), Point2d(0, 0)});
+  Polygon polygon = GoalRectangle();
```

Comment:
Ah okay, the GoalRectangle shall simplify the code. But @cirrostratus1  is right, the current definition does not support the normal car shape. At least the Goal Rectangle should have parameters, to specify width and height.

Reply:
solved